src/utils.py

```diff
@@ -0,0 +1,16 @@
+from src.base import IndexerInput
+
+
+def filter_on_block_type(inputs: list[IndexerInput], remove_block_types: list[str]) -> list[IndexerInput]:
+    """Filter a sequence of IndexerInputs to remove the textblocks that are of the types declared in the remove block
+    types array. """
+
```

Comment:
Can we `block.title() for block in remove_block_types` or even check they're in the enum here? Maybe a good idea as they're set from env variables